libs/ledger-live-common/src/families/elrond/js-signOperation.ts

```diff
@@ -139,16 +140,17 @@ const signOperation: SignOperationFnSignature<Transaction> = ({ account, deviceI
 
           if (tokenAccount) {
             const { token } = decodeTokenAccountId(tokenAccount.id);
-
-            if (token?.name && token.id && token.ledgerSignature) {
-              await elrond.provideESDTInfo(
-                token.name,
-                extractTokenId(token.id),
-                token?.units[0].magnitude,
-                CHAIN_ID,
-                token.ledgerSignature,
-              );
+            if (token == null) {
+              throw new InvalidToken();
```

Comment:
If my understanding is correct, this can happen to users with existing token accounts, right?

We are globally trying to improve our user-facing errors and messages, so they make more sense and so the UX is better.
Here's a proposal that allows to provide some more info to the user, please set a message that makes sense:
```suggestion
              throw new Error("message here");
```